core/src/main/java/org/springframework/security/authentication/dao/DaoAuthenticationProvider.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004, 2005, 2006 Acegi Technology Pty Limited
+ * Copyright 2002-2020 the original author or authors.
```

Comment:
Will you please leave the `Acegi` copyright headers as-is?

Reply:
done